plugin-api.d.ts

```diff
@@ -1369,6 +1369,7 @@ declare type Trigger =
   | {
       readonly type: 'MOUSE_ENTER' | 'MOUSE_LEAVE' | 'MOUSE_UP' | 'MOUSE_DOWN'
       readonly delay: number
+      readonly deprecatedVersion: boolean
```

Comment:
I think it would be preferable to split up the union types between Enter/Leave and Up/Down since there should never be a deprecatedVersion field on the latter